Mathlib/Order/OrderType.lean

```diff
@@ -0,0 +1,450 @@
+/-
+Copyright (c) _
+-/
+import Mathlib.SetTheory.Cardinal.Basic
+import Mathlib.SetTheory.Ordinal.Basic
```

Comment:
We already talked about this, but IMO ordinals should import order types, not the other way around